packages/techdocs-common/src/stages/publish/local.test.ts

```diff
@@ -59,40 +42,34 @@ const logger = getVoidLogger();
 describe('local publisher', () => {
   it('should publish generated documentation dir', async () => {
     const testDiscovery: jest.Mocked<PluginEndpointDiscovery> = {
-      getBaseUrl: jest.fn().mockResolvedValue('http://localhost:7000'),
+      getBaseUrl: jest
+        .fn()
+        .mockResolvedValue('http://localhost:7000/api/techdocs'),
       getExternalBaseUrl: jest.fn(),
     };
 
-    const mockConfig = new ConfigReader({
-      techdocs: {
-        requestUrl: 'http://localhost:7000',
-        storageUrl: 'http://localhost:7000/static/docs',
-      },
-    });
+    const mockConfig = new ConfigReader({});
 
     const publisher = new LocalPublish(mockConfig, logger, testDiscovery);
     const mockEntity = createMockEntity();
     const tempDir = fs.mkdtempSync(`${__dirname}/test-component-folder-`);
```

Comment:
this one is a pretty big wtf to me :) should probably use mockFs

Reply:
Haha, yes indeed. Every time the test fails, the directory remains uncleaned. :P